### PR TITLE
Dox42

### PR DESCRIPTION
**Pull request by dox42diegoschleis**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/dox42/apiDefinition.swagger.json [hunk 1]
```diff
@@ -26,8 +26,8 @@
             "schema": {}
           }
         },
-        "summary": "dox42 Service Call",
-        "description": "Call to the dox42 service",
+        "summary": "dox42 Service Call GET",
+        "description": "Make an HTTP GET request to the dox42 service",
         "operationId": "dox42_Call",
```
Comment (v-EgorKozhadei) [inline certified-connectors/dox42/apiDefinition.swagger.json:31]: @dox42diegoschleis operationId properties should [use pascal casing](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers). Please make a quick fix for operationId properties.

File: certified-connectors/dox42/apiDefinition.swagger.json [hunk 2]
```diff
@@ -56,6 +56,53 @@
             "x-ms-summary": "Basic accept header, leave as is!"
           }
         ]
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (v-EgorKozhadei) [inline certified-connectors/dox42/apiDefinition.swagger.json:64]: @dox42diegoschleis empty response schemas and default responses with an empty schema are not allowed. Could you please define successful 2xx response and a schema? An example of responses can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

File: certified-connectors/dox42/apiDefinition.swagger.json [hunk 3]
```diff
@@ -56,6 +56,53 @@
...
+          }
+        },
+        "summary": "dox42 Service Call POST",
+        "description": "Make an HTTP POST request to the dox42 service",
+        "operationId": "dox42_Call_POST",
```
Comment (v-EgorKozhadei) [inline certified-connectors/dox42/apiDefinition.swagger.json:69]: @dox42diegoschleis mentioned in the comment above, please use pascal case here.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @dox42diegoschleis! 

Thank you for an update!

Please check comments after the review, there are a few minor things that need to be fixed.

Let me know if you have any questions!

File: certified-connectors/dox42/apiDefinition.swagger.json (hunk 1 above)
Comment (dox42diegoschleis) [reply certified-connectors/dox42/apiDefinition.swagger.json]: I have changed the operationId to pascal casing

File: certified-connectors/dox42/apiDefinition.swagger.json (hunk 3 above)
Comment (dox42diegoschleis) [reply certified-connectors/dox42/apiDefinition.swagger.json]: I have changed the operationId to pascal casing

File: certified-connectors/dox42/apiDefinition.swagger.json (hunk 2 above)
Comment (dox42diegoschleis) [reply certified-connectors/dox42/apiDefinition.swagger.json:70]: I have now defined a 200 response, our service will not return 4xx or 5xx by default, so defining them is out of the question

Comment (v-EgorKozhadei) [reply certified-connectors/dox42/apiDefinition.swagger.json:70]: Could you please change the default response to 200 response and remove an empty schema in this case? If you want to keep the schema, you can define it like string type in your other operations.

Comment (dox42diegoschleis) [reply certified-connectors/dox42/apiDefinition.swagger.json:66]: Changed

**Comment by v-EgorKozhadei** [comment]
Hi @dox42diegoschleis!

Your PR is approved and merged. Please update your submission in ISV Studio by using the latest commit ID.

Let me know if you have any questions!

**Pull request merged**